### ISSUE TITLE
RCA issues faced by RedHat

### ISSUE THREAD

**Issue by kerneltime** [vmware/kubernetes#85]

**Comment by kerneltime** [comment]
- Had a misconfigured master  (This seems resolved)
- Request failing in VC (on going)

**Comment by kerneltime** [comment]
Reported the fixes part of 1.5.3
```
There 2 commits in the pipeline that should be part of 1.5.3
 
https://github.com/kubernetes/kubernetes/pull/41217 (Requires changes in config please read the PR and the updated docs for getting started on vSphere)
https://github.com/kubernetes/kubernetes/pull/41226
 
Changes that are part of 1.5 branch but not in 1.5.2
 
Cloud provider:
 
https://github.com/kubernetes/kubernetes/commit/e14a92de8a584eae4e71cfbcb12046a52b65719a#diff-b0a0aeb7e17950041ec1d5d28a251826
https://github.com/kubernetes/kubernetes/commit/13d9c8bcd03c8bbba5df666f54edf866cff8041f#diff-b0a0aeb7e17950041ec1d5d28a251826
https://github.com/kubernetes/kubernetes/commit/74442bb029df2f83ed3b0003c55a12bbdc4fd759#diff-b0a0aeb7e17950041ec1d5d28a251826
 
Volume:
https://github.com/kubernetes/kubernetes/pull/39757
https://github.com/kubernetes/kubernetes/pull/39754
 
One easy way to figure this out is to look at what the last tag vs the branch is for the cloud provider and the volume
 
Tag
https://github.com/kubernetes/kubernetes/commits/v1.5.2/pkg/volume/vsphere_volume
https://github.com/kubernetes/kubernetes/commits/v1.5.2/pkg/cloudprovider/providers/vsphere
 
vs
 
Branch
https://github.com/kubernetes/kubernetes/commits/release-1.5/pkg/volume/vsphere_volume
https://github.com/kubernetes/kubernetes/commits/release-1.5/pkg/cloudprovider/providers/vsphere
```

**Comment by pdhamdhere** [comment]
@kerneltime There is no AI on this one. Closing.
